.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "extends": "airbnb",
+    "parserOptions": {
+      "ecmaVersion": 6
+      },
+      "env": {
+        "node": true,
+        "browser": true,
+        "es6": true
+        },
+        "rules": {
+          "react/jsx-no-bind": ["error", {
+          "ignoreRefs": true,
+          "allowArrowFunctions": true,
+          "allowFunctions": true,
+          "allowBind": true,
+          "ignoreDOMComponents": true
+          }],
+          "comma-dangle": "off",
+          "import/prefer-default-export": "off",
+          "no-unused-vars": 0, 
+           "no-console": "off",
```

Comment:
Does this mean ESLint will allow console.log()?

Reply:
@nignanthomas i did a mistake here on no-console let me correct it and some rules